### ISSUE TITLE
Closing lid suspends laptop when external monitor is connected.

### ISSUE THREAD

**Issue by hgaiser** [pop-os/cosmic-epoch#307]
I'm used to connecting an external monitor and closing my laptop, however with Cosmic this makes the laptop suspend. I don't think there is an option for changing this behavior (yet?), but I think it makes sense for the default to be to not suspend the laptop.

**Comment by jokeyrhyme** [comment]
Yeah, I think GNOME and other desktops intercept such systemd/ACPI events

**Comment by jokeyrhyme** [comment]
According to the latest comment here we can probably close this issue: https://github.com/pop-os/cosmic-comp/issues/19

**Comment by hgaiser** [comment]
Ideally it would also disable the closed laptop screen, but for now we can consider this fixed I suppose :).

Thanks for letting me know.

**Comment by bobcat141** [comment]
> According to the latest comment here we can probably close this issue: [pop-os/cosmic-comp#19](https://github.com/pop-os/cosmic-comp/issues/19)

Is this already available in the COSMIC alpha? Closing the lid still puts my laptop to sleep even with an external monitor plugged in. Maybe it's because it's plugged in through a USB-C cable?

**Comment by hgaiser** [comment]
It doesn't seem to work still, not on alpha1, nor on the latest git version. Reopening for now :shrug:

**Comment by bartv42** [comment]
My experience is different: closing the laptop (Thinkpad X13) keeps it running, but doesn't disable the built-in screen.

**Comment by guzman109** [comment]
> My experience is different: closing the laptop (Thinkpad X13) keeps it running, but doesn't disable the built-in screen.

I have the same issue. I have to disable it manually.

**Comment by iamkartiknayak** [comment]
> It doesn't seem to work still, not on alpha1, nor on the latest git version. Reopening for now 🤷

The laptop suspending problem seems to have been resolved in `popdev:master` so it'll eventually be released to all systems. Now the bug seems to be keeping the primary monitor enabled even when lid is closed. Can you consider changing the title?

**Comment by hgaiser** [comment]
Doesn't it make more sense to open a new issue for that?

**Comment by iamkartiknayak** [comment]
> EDIT: For clarity, my laptop suspends regardless of a connected (HDMI) monitor.

Do you still have this same issue?

**Comment by hgaiser** [comment]
I can't test it unfortunately as I'm on vacation.. I'll get back to you in
two weeks, unless I forget 😇

**Comment by cart** [comment]
On Cosmic Alpha 3 my laptop still suspends when the lid is closed and I have an external monitor plugged in (via a USB-C hub). Works as expected (staying awake on close when the external monitor is plugged in) in Plasma 5.

**Comment by bobcat141** [comment]
I've had the same behavior as @cart, with the addition that the external monitor doesn't turn on again after I wake up the laptop.

**Comment by auggie246** [comment]
Issue still persist in alpha 6, Connected to a display via usb-c, system still goes to sleep when laptop lid is closed

**Comment by jacobgkau** [comment]
Related issue in the cosmic-comp repo: https://github.com/pop-os/cosmic-comp/issues/1351

**Comment by auggie246** [comment]
Seems to be working now. Though I also did the following via `sudo vim /etc/systemd/logind.conf`

```
HandleLidSwitchExternalPower=ignore
HandleLidSwitchDocked=ignore
```

**Comment by bobcat141** [comment]
It's working for me too out of the box with USB-C.
